cherrypy/test/test_states.py

```diff
@@ -161,6 +161,11 @@ def test_1_Restart(self):
         self.assertEqual(db_connection.running, False)
         self.assertEqual(len(db_connection.threads), 0)
 
+    @pytest.mark.forked
+    @pytest.mark.xfail(
+        bool(os.getenv('CI')),
+        reason='keyboard interrupts fail on continuous integration'
```

Comment:
Plz add https://github.com/cherrypy/cherrypy/issues/1873 ref to the reason string.

Also, this requires a hack https://github.com/pytest-dev/pytest-forked/issues/44 for windows that doesn't have fork.

Reply:
And maybe mention that it's probably a regression caused by https://github.com/cherrypy/cheroot/pull/309 too

Reply:
Please put the cherrypy/cheroot#309 reference in #1873.